python/lsst/ctrl/bps/generic_workflow.py

```diff
@@ -394,21 +395,36 @@ def quanta_counts(self):
                 qcounts += gwjob.quanta_counts
         return qcounts
 
+    @property
+    def labels(self):
+        """List of job labels"""
+        return list(self._labels.keys())
+
+    def regenerate_labels(self):
+        """Regenerate the list of job labels.
+
+        Returns
+        -------
+        labels : `list` [`str`]
```

Comment:
The description of the return value is missing.

Reply:
Decided to remove return value and that section from docstring.